### ISSUE TITLE
select_company

### ISSUE THREAD

**Issue by bruced** [davidlumley/myob-api#20]
no implicit conversion of String into Integer
@api_client.select_company_file({:id => "7c13eeb8-snip",:username => "snip",:password=>"snip",
})

Any ideas on what is wrong here

**Comment by karpit** [comment]
Yes, If you are trying to get all company files in below manner like  -
@api_client.company_file.all_items
Then Might be you are facing such type of error but If you write in this manner @api_client.company_file.all 
then no such error you will get.
